README.md

```diff
@@ -21,10 +40,11 @@ The current release version is 1.2.0
 
 Ensure the development version of the dependencies are installed. These are as follows:
 
-* HDB++ library libhdbpp
+* HDB++ library libhdbpp (downloaded automatically by cmake if not found)
 * Tango Controls 9 or higher - either via debian package or source install.
 * omniORB release 4 - libomniorb4 and libomnithread.
 * libzmq - libzmq3-dev or libzmq5-dev.
+* MySQL client library - libmysqlclient-dev or libmariadb-dev
```

Comment:
shouldn't it be libmariadbclient-dev ?
Testing on debian9 where libmysqlclient-dev is not available, it is leading to some trouble cause on compilation it complains about TYPE_MYSQL_JSON not being defined, the reason being that support for it is added on mariadb 10.2.3 (https://mariadb.com/kb/en/json_type/). No need to say that this version is not the one in the debian repos…

Reply:
OK, actually I am using ubuntu 18.04 with libmysqlclient-dev or debian 10 with libmariadb-dev (10.3)

Reply:
Just checked that in debian 9 package was called libmariadbclient-dev, while in debian  >= 10 libmariadbclient-dev is empty and the right one is libmariadb-dev

Reply:
If even debian is adding to the confusion, what will we do !? In the end the name of the package itself is not so much of a problem, but we should have the minimum required version for sure.

Reply:
Agree, since old version are not supported, I will leave the new name with the minimum required version.